daily-triage/tableflip-triage-email.sh

```diff
@@ -155,13 +160,18 @@ subject="Daily triage for: $projects [$triager]"
             echo "<ul>"
             cat "$project-reviews.html"
             echo "</ul>"
+            rm -f "$project-reviews.html"
         fi
     done
 
-    echo "<h5>Yesterday's top 5 subiquity crashes</h5>"
-    echo "<ul>"
-    cat subiquity_crashes.html
-    echo "</ul>"
+    for github_project in "${github_projects[@]}"; do
+        if [ -e "$github_project-reviews.html" ]; then
```

Comment:
This now exists always (and contains "None")  to we want to check that it is not None? Befor emitting the specific github project section at all should we be checking that the file doesn't contain None or do we always want this section to print.
```suggestion
        if [ "None" != "$( cat $github_project-reviews.html)" ]; then
```

Reply:
I think a better solution is to delete the file if there are no reviews and leave the `[ -e` test here (and in the code that generated the plaintext email).